### ISSUE TITLE
shareable URLs

### ISSUE THREAD

**Issue by trusktr** [shawnlawson/The_Force#35]
That would be neat!

**Comment by shawnlawson** [comment]
I agree. I think Hydra [https://github.com/ojack/hydra](https://github.com/ojack/hydra) does with with a string hash on the URL. Will check on it.
